src/components/HomePage/index.js

```diff
@@ -180,17 +178,18 @@ export default class HomePage extends React.Component{
                 <MenuItem onClick={() => this.handleClose("Project")}>Project</MenuItem>
                 <MenuItem onClick={() => this.handleClose("None")}>None</MenuItem>
                 </Menu>
-
+                <Fab color="primary" aria-label="add" className="fab-icon">
+                    <Link href="/new-post" variant="body2">
```

Comment:
Wrapping Link over the button better IMO, because there were cases where it took me to user profile, because I didn't click on the '+' exactly